shell/components/Drawer/ResourceDetailDrawer/composables.ts

```diff
@@ -13,12 +13,13 @@ export function useResourceDetailDrawer() {
       returnFocusSelector,
       {
         resource,
-        onClose:   close,
-        width:     '73%',
+        onClose:            close,
+        width:              '73%',
         // We want this to be full viewport height top to bottom
-        height:    '100vh',
-        top:       '0',
-        'z-index': 101 // We want this to be above the main side menu
+        height:             '100vh',
+        top:                '0',
+        'z-index':          101, // We want this to be above the main side menu
+        closeOnRouteChange: ['name', 'params', 'query'] // We want to ignore hash changes, tables in extensions can trigger the drawer to close while opening
```

Comment:
The tables from the harvester config pages were causing the drawer to close. In Rancher I just disabled the use of hash use of Tabs but since that change can't be done for extensions I decided to make this change instead.